### DISCUSSION TITLE
请问申请学位类别和学位类别选项如何分开填写

### DISCUSSION THREAD

**Discussion by Jamie-obj** [note286/xduts#141] [Q&A] [answered]
学院要求如下：
①论文封面中“申请学位类别”，在下拉选项中，根据自己情况选择‘电子信息硕士’或‘机械硕士’
②其他地方的“学位类别”写：‘专业学位’，和你们系统中的写法都是一致的（实在改不了的，写专业学位工程硕士）
但是我设置封面的申请学位类别和第二页的学位类别处都是由`info / degree`这个选项控制的，请问我该如何分开选择填写呢

**Comment by note286** [comment, marked as answer]
目前完全根据学位办要求设计。

目前你如果有特殊需求，手动修改cls吧，或者直接PDF编辑器修改。

**Reply by Jamie-obj** [reply]
好的👌

**Reply by note286** [reply]
你是哪个学院的，我头一次听说西电还有学院有自己要求的。

**Comment by Jamie-obj** [comment]
> 你是哪个学院的，我头一次听说西电还有学院有自己要求的。

广研院的，电子信息第一届毕业，搞得特别乱。。

**Reply by note286** [reply]
学位办给的Microsoft Word模板是下拉的，当然，你可以强制修改。

![image](https://github.com/note286/xduts/assets/16207292/a6a9789b-e0b9-4f40-a21a-b42b0d733cd0)

**Reply by Jamie-obj** [reply]
感谢，修改cls文件更方便✌
